sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/ChangeFeedProcessorItem.java

```diff
@@ -69,7 +89,18 @@ public ChangeFeedMetaData getChangeFeedMetaData() {
      */
     @Beta(value = Beta.SinceVersion.V4_37_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
     public JsonNode toJsonNode() {
-        return Utils.getSimpleObjectMapper().convertValue(this, JsonNode.class);
+
+        if (this.changeFeedProcessorItemAsJsonNode == null) {
+            this.changeFeedProcessorItemAsJsonNode = Utils.getSimpleObjectMapper().convertValue(this, JsonNode.class);
```

Comment:
We don't need to convert, we can create an ObjectNode and then set current, previous and metadata.

Reply:
Fixed in next iteration.